### PR TITLE
replaces calls to obsolete route transition method with supported equivalent

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes https://github.com/ilios/ilios/issues/4952

### CODE REVIEW

File: app/components/ilios-users.hbs [hunk 1]
```diff
@@ -65,14 +65,15 @@
...
     {{#if this.searchForUsers.lastSuccessful}}
-      <PagedlistControls
-        @offset={{@offset}}
-        @limit={{@limit}}
-        @limitless={{true}}
-        @setOffset={{@setOffset}}
-        @setLimit={{@setLimit}}
-        data-test-top-paged-list-controls
-      />
+      <div data-test-top-paged-list-controls>
```
Comment (jrjohnson) [inline app/components/ilios-users.hbs:68]: I don't know why, but I'm sort of bothered by doing a div wrap just for testing. I don't see an easy way to target the element even if we pass the `data-test-` into the component though so let's go with it, mainly it's just late and I'm ornery about all code today.

File: app/components/new-user.js [hunk 2]
```diff
@@ -49,6 +49,7 @@ export default class NewUserComponent extends Component {
   get schoolsWithCreatePermissions() {
     return new TrackedAsyncData(
       filter(this.allSchools, async (school) => {
+        console.log(school);
```
Comment (jrjohnson) [inline app/components/new-user.js:52]: `//be cool console log school`

**Review by jrjohnson — CHANGES_REQUESTED** [review]

File: app/components/ilios-users.hbs (hunk 1 above)
Comment (stopfstedt) [reply app/components/ilios-users.hbs:68]: i know, i was annoyed by that too.

File: app/components/new-user.js (hunk 2 above)
Comment (stopfstedt) [reply app/components/new-user.js:52]:
```suggestion
```

**Review by jrjohnson — APPROVED** [review]

**Pull request merged**